src/devtools/domains/history.js

```diff
@@ -33,12 +36,18 @@ class History {
       }
     }
 
-    return { ...newHistory, list }
+    return { ...newHistory, list, focused }
+  }
+
+  focus(old, id) {
+    let focused = id
+    return { ...old, focused }
   }
```

Comment:
Style nitpick: what do you think of naming the second argument `focused` and nixing line 43? Like:

```
focus (old, focused) {
  return { ...old, focused }
}
```